jobs/build.Jenkinsfile

```diff
@@ -94,6 +95,7 @@ pod = pod.replace("COREOS_ASSEMBLER_CPU_REQUEST", "4")
 // substitute the right COSA image and mem request into the pod definition before spawning it
 pod = pod.replace("COREOS_ASSEMBLER_MEMORY_REQUEST", "${cosa_memory_request_mb}Mi")
 pod = pod.replace("COREOS_ASSEMBLER_IMAGE", params.COREOS_ASSEMBLER_IMAGE)
+pod = pod.replace("JENKINS_AGENT_IMAGE", params.JENKINS_AGENT_IMAGE)
```

Comment:
Do we mean to use `jenkins_agent_image`? `params.JENKINS_AGENT_IMAGE` is `null`

Reply:
Fixed!

Reply:
OK, let's switch to a higher bandwidth channel to debug this.